app/src/lib/tokens/Word.svelte

```diff
@@ -1,18 +1,15 @@
 <script>
-	import {TOKEN_TYPE, token_has_concept} from '$lib/parser/token'
-	import Error from './Error.svelte'
+	import {token_has_concept} from '$lib/parser/token'
 	import NotFound from './NotFound.svelte'
 	import Result from './Result.svelte'
 
 	/** @type {Token} */
 	export let token
-	export let classes = ''
+	export let should_join = false
```

Comment:
I'm thinking this may not be necessary since `should_join` is optional (and defaulted to false already).

Reply:
Changed back to using 'classes'